app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -44,11 +45,20 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   searchResult: {
-    [theme.breakpoints.down("sm")]: {
-      maxWidth: "80%",
-    },
-    [theme.breakpoints.up("md")]: {
-      margin: theme.spacing(2),
+    marginBottom: theme.spacing(3),
+    borderRadius: theme.shape.borderRadius * 2,
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: "0 0 4px rgba(0,0,0,0.25)",
+    [theme.breakpoints.down("md")]: {
+      "& > button": {
+        height: "100%",
+      },
+      "& > button > *": {
+        padding: 0,
+        margin: theme.spacing(2),
+        height: "calc(100% - 2rem)",
```

Comment:
```suggestion
        height: `calc(100% - ${theme.spacing(1)})`,
```
However I wonder if it's easier for the parent to have a padding, and the height to just be 100%. Or even better, `alignItems: "stretch"`?